LenovoLegionToolkit.WPF/Controls/ColorCardControl.cs

```diff
@@ -14,6 +15,8 @@ namespace LenovoLegionToolkit.WPF.Controls
 {
     public class ColorCardControl : UserControl
```

Comment:
This control is also used for RGB keyboard controls so it's not a good idea to add the system accent color checkbox here.

Reply:
> 

Interesting. I don't have a Lenovo machine. Does the Accent Color setting in the Color Card Control also set any colors for the Keyboard?

I'll try to separate it from the color card control and add something probably directly under the Theme settings for App Accent Color?